crates/hir-expand/src/fixup.rs

```diff
@@ -276,6 +275,62 @@ pub(crate) fn fixup_syntax(
                         ]);
                     }
                 },
+                ast::RecordExprField(it) => {
```

Comment:
sorry, dumb question: why are the fixup changes included in this PR?

Reply:
Because I was too lazy to put them into a separate PR earlier and only later realized I won't self-merge this PR immediately as I usually do